### PR TITLE
russian localization updates and fixes

### PR DESCRIPTION
**Pull request by Emptyfruit**
fixed some typos
added some of the missing translations (important ones)
organized stream.ttl to match the EN version in lines and sections (mostly) for future edits convenience.

### CODE REVIEW

**Comment by Emptyfruit** [comment]
If you don't mind me asking, where 6.6.0 can be expected to be released? This PR is vital for usage, some very basic things were messed up, including submit button. Is possible, may be this can be moved to 6.4.* minor or at least 6.5 ? I did not dare to make a PR to master, but really need these fixes asap..

**Comment by wyattjoh** [comment]
Are you referring to the localization changes specifically? There's a chance we can release this change directly if it's all that's blocking you!

**Comment by Emptyfruit** [comment]
Exactly, only these changes to localization. Appreciate it.

File: src/locales/ru/stream.ftl [hunk 1]
```diff
@@ -212,7 +247,19 @@ comments-rejectedTombstone = Вы отклонили этот комментар
...
+  .aria-label = {$count ->
+    [0] {$reaction} comment by {$username}
+    *[other] {$reaction} ({$count}) comment by {$username}
+  }
+comments-reacted =
+  .aria-label = {$count ->
+    [0] {$reaction} comment by {$username}
+    [one] {$reaction} comment by {$username}
+    *[other] {$reaction} ({$count}) comment by {$username}
+  }
```
Comment (wyattjoh) [inline src/locales/ru/stream.ftl]: You should not include english text.

File: src/locales/ru/stream.ftl [hunk 2]
```diff
@@ -261,34 +315,61 @@ qa-answeredTooltip-answeredComments =
...
-
 profile-myCommentsTab = Мои комментарии
 profile-myCommentsTab-comments = Мои комментарии
 profile-accountTab = Аккаунт
 profile-preferencesTab = Настройки
-
 accountSettings-manage-account = Настройте свой аккаунт
 
+### Bio
+profile-bio-title = Bio
```
Comment (wyattjoh) [inline src/locales/ru/stream.ftl]: Other english text.

**Review by wyattjoh — CHANGES_REQUESTED** [review]

File: src/locales/ru/stream.ftl [hunk 3]
```diff
@@ -50,37 +50,66 @@ comment-count-text =
...
 comments-replyList-showAll = Показать все
 comments-replyList-showMoreReplies = Показать больше ответов
 
+comments-postCommentForm-gifSeach = Искать GIF
+comments-postComment-gifSearch-search =
+  .aria-label = Искать
+comments-postComment-gifSearch-loading = Поиск...
+comments-postComment-gifSearch-no-results = Ничего не найдено по «{$query}»
+comments-postComment-gifSearch-powered-by-giphy =
+  .alt = Powered by giphy
```
Comment (wyattjoh) [inline src/locales/ru/stream.ftl]: English should be removed here as well.

**Review by kgardnr — APPROVED** [review]

**Review by wyattjoh — APPROVED** [review]

**Comment by Emptyfruit** [comment]
If you don't mind ill raise the question again: can these changes be released really soon? 6.6.0 has no release date yet, but we really need it soon.

**Comment by Emptyfruit** [comment]
Sorry, guys, once again, our deadline is in a week. I see that 6.6.0 is almost done but has no release date. May be my changes can be merged into 6.5.1 ?

**Comment by wyattjoh** [comment]
We're making a release this week with all these in it! Expect the release to go out by Wednesday!

**Pull request merged**